lib/go-tc/invalidationjobs.go

```diff
@@ -349,12 +356,13 @@ func (job *UserInvalidationJobInput) Validate(tx *sql.Tx) error {
 
 	if job.TTL != nil {
 		row := tx.QueryRow(`SELECT value FROM parameter WHERE name='maxRevalDurationDays' AND config_file='regex_revalidate.config'`)
-		var max uint64
-		err := row.Scan(&max)
+		var maxDays uint64
+		err := row.Scan(&maxDays)
+		maxHours := maxDays * 24
 		if err == sql.ErrNoRows && MaxTTL < *(job.TTL) {
 			errs = append(errs, "ttl: cannot exceed "+strconv.FormatUint(MaxTTL, 10)+"!")
-		} else if err == nil && max < *(job.TTL) { //silently ignore other errors to
-			errs = append(errs, "ttl: cannot exceed "+strconv.FormatUint(max, 10)+"!")
+		} else if err == nil && maxHours < *(job.TTL) { //silently ignore other errors to
+			errs = append(errs, "ttl: cannot exceed "+strconv.FormatUint(maxHours, 10)+"!")
```

Comment:
Errors should not end with punctuation

Reply:
done in bae7ca548739e0d9fe501afba8b2c2559b7843f9